README.md

```diff
@@ -29,6 +29,9 @@ Before using the core package, see if one of the options above will work for you
     <version>${work-tracker.version}</version>
 </dependency>
 ```
+### Module support for Java 9 and later
+`requires com.deere.isg.worktracker.core;`
+See [example](./work-tracker-servlet/src/main/java9/module-info.java)
```

Comment:
This should come **after** the pins statement below.  See the doc on outstanding for how a little more wording here would make this part of the documentation for friendly.

Reply:
I'd like to move this ## Installation section to a new README.md in work-tracker-core.  Then link there from the ## Setup section.  Mention core last in the list, saying use it directly only if one of the other options does not apply. 

I can make this change post-merge, if you prefer.